compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -453,7 +453,7 @@ execIde telemetry (Debug debug) enableScenarioService options =
                   then Logger.Debug
                   -- info is used pretty extensively for debug messages in our code base so
                   -- I've set the no debug threshold at warning
```

Comment:
This comment should be updated or removed.

Reply:
good catch, just dropped it. I think INFO is the usual default so doesn’t really need an explanation.